### ISSUE TITLE
PropNetEstimator causes PyTorch "UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`"

### ISSUE THREAD

**Issue by MrMois** [nerfstudio-project/nerfacc#289]
Hello, thanks for this awesome library!

When using the PropNetEstimator, e.g. by running the example script, pytorch throws the warning

> UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate

This is probably caused by [](https://github.com/nerfstudio-project/nerfacc/blob/32273f838184f4c6345b8371aaa17f2a13d62adf/nerfacc/estimators/prop_net.py#L177). It doesn't interfere with training, but I think it would be good to prevent any warnings thrown by using nerfacc as intended.

Best,
MrMois
